bootstrap/kubeadm/api/v1alpha2/conversion.go

```diff
@@ -129,3 +129,8 @@ func Convert_v1alpha2_KubeadmConfigSpec_To_v1alpha3_KubeadmConfigSpec(in *Kubead
 func Convert_v1alpha3_KubeadmConfigSpec_To_v1alpha2_KubeadmConfigSpec(in *kubeadmbootstrapv1alpha3.KubeadmConfigSpec, out *KubeadmConfigSpec, s apiconversion.Scope) error {
 	return autoConvert_v1alpha3_KubeadmConfigSpec_To_v1alpha2_KubeadmConfigSpec(in, out, s)
 }
+
+func Convert_v1alpha3_File_To_v1alpha2_File(in *kubeadmbootstrapv1alpha3.File, out *File, s apiconversion.Scope) error {
+	// We don't implement manual conversion for types using contentFrom
+	return autoConvert_v1alpha3_File_To_v1alpha2_File(in, out, s)
+}
```

Comment:
We should still restore it when we go `v1alpha3 -> v1alpha2 -> v1alpha3`, there is an example in other files, or I can help tomorrow

Reply:
will add